### PR TITLE
Merge to Deploy 

### PR DESCRIPTION
**Pull request by thesifter**
:shipit:

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -7,7 +7,12 @@ import Menu from './Menu'
 class Dashboard extends Component {
   constructor (props) {
     super(props)
-    this.state = {modal: false}
+    this.state = {
+      modal: false,
+      contact: 'sms://+14035550185',
+      vulnerableMessage: '?body=I%20am%20feeling%20vulnerable',
```
Comment (amandahogan) [inline src/components/Dashboard.js:14]: you may have to do a if else statement here ? and : to make it work on both iphone and android... iphone changes the ? into & ... ask zane, he figured it out yesterday.

**Review by amandahogan — CHANGES_REQUESTED** [review]
Hey if you want to test it on your phone before deploying into master you can run the command `yarn build` and then `serve -s build` you might have to install serve globally first if you don't have it `yarn global add serve` and then on your phone type in the url that comes up on your command line for On Your Network. make sure your phone is on the same wifi as your computer.

**Comment by jeremy-liss** [comment]
oh cool, thanks. Will try that

File: src/components/Dashboard.js (hunk 1 above)
Comment (thesifter) [reply src/components/Dashboard.js:14]: Thanks! 👍

**Pull request merged**